apps/web/src/components/SpeakersList/SpeakersList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { SpeakerMenu } from './SpeakerMenu/SpeakerMenu';
+import { useSpeakersMap } from '../../hooks/useSpeakersMap/useSpeakersMap';
+import useVideoContext from '../../hooks/useVideoContext/useVideoContext';
+
+const useStyles = makeStyles({
+  header: {
+    fontWeight: 'bold',
+    padding: '1em 1em 0',
+  },
+  speakersListContainer: {
+    padding: '0.4em 1em',
+    borderBottom: '0.1em solid #CACDD8',
+    '& p': {
+      padding: '0.5em 0',
+    },
+  },
+  speakerContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '100%',
+    alignItems: 'center',
+  },
+});
+
+export default function SpeakersList() {
+  const { host, speakers } = useSpeakersMap();
+  const { room } = useVideoContext();
+  const localParticipant = room!.localParticipant;
```

Comment:
Just had a thought - the ParticipantWindow should be available to Viewers too right? In addition to the Host and Speakers? 

Feels like it should be easy to do, but I think we'd need to handle the case for Viewers where `room` is `null`.

Reply:
Looks like all that was needed was some optional chaining! I also added the participant window to the `<PlayerMenuBar />` component. I decided to add a new simple `toggleParticipantWindow()` function since the `ToggleParticipantWindowButton` component contained a lot more functionality than what the Viewer needs.
![Screen Shot 2022-01-05 at 1 14 43 PM](https://user-images.githubusercontent.com/77076398/148291578-e2e49493-27f9-42d0-944b-5000cc3f6194.png)